R/parse_expr.R

```diff
@@ -41,6 +41,14 @@ parse_expr_assignment <- function(expr, src, call) {
         "E1014", src, call)
     }
     special <- "parameter"
+  } else if (rhs$type == "interpolate") {
+    if (!is.null(special)) {
+      odin_parse_error(
+        "Calls to 'interpolate()' must be assigned to a symbol",
+        "E1099", src, call)
```

Comment:
E1099 not implemented yet - bit like E1035 but for lhs...

Reply:
thanks, these should be fixed now